src/ast/visitor.ts

```diff
@@ -169,6 +172,9 @@ export abstract class ASTVisitor<T> {
   visitCairoFunctionDefinition(node: CairoFunctionDefinition, ast: AST): T {
     return this.visitFunctionDefinition(node, ast);
   }
+  visitCairoGeneratedFunctionDefinition(node: CairoGeneratedFunctionDefinition, ast: AST): T {
```

Comment:
`CairoImportFunctionDefintion` is used as a placeholder for external (outside of the sourceUnit) functions. Functions which don't use anything from the current source unit. So they get the default visitor, and since they have an empty body, nothing besides the signature gets read.